qutebrowser/javascript/webelem.js

```diff
@@ -297,33 +274,24 @@ window._qutebrowser.webelem = (function() {
             return null;
         }
 
-        // Check if we got an iframe, and if so, recurse inside of it
-        const frame_elem = call_if_frame(elem,
-            (frame) => serialize_elem(frame.document.activeElement, frame));
-
-        if (frame_elem !== null) {
-            return frame_elem;
-        }
-        return serialize_elem(elem);
+        // Get the window of the frame/root
+        const frame_win = utils.get_frame_window(elem);
+        return serialize_elem(frame_win.document.activeElement, frame_win);
     };
 
     funcs.find_at_pos = (x, y) => {
         const elem = document.elementFromPoint(x, y);
 
-
         // Check if we got an iframe, and if so, recurse inside of it
```

Comment:
doesn't seem to be actually recursive (is a frame inside a frame handled?)

Reply:
In theory it should be (there could be e.g. an iframe in a frame), but I'd also be okay with just adjusting the comment to match the implementation.